common/tcbinfo.c

```diff
@@ -509,6 +509,201 @@ oe_result_t oe_parse_tcb_info_json(
     return result;
 }
 
+OE_INLINE uint32_t ReadUint32(const uint8_t* p)
+{
+    return p[0] | (p[1] << 8) | (p[2] << 16) | (p[3] << 24);
+}
+
+OE_INLINE uint64_t ReadUint64(const uint8_t* p)
+{
+    uint64_t temp = 0;
+    uint64_t result = 0;
+    for (int i = 7; i >= 0; i--)
+    {
+        temp = p[i];
+        result = (result << 8) | temp;
```

Comment:
Revers order?

Reply:
The string field in json stream comes in reverse order.

Reply:
Would be good to document the rationale for this reverse ordering.